examples/build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.3'
+    }
+}
+
+ext {
+    defaultCompileSdkVersion = 25
+    defaultMinSdkVersion = 9
```

Comment:
16, not 9. Less than 3% are on 15 and below (https://developer.android.com/about/dashboards/index.html)